### PR TITLE
Can see his summary his cart: see, add, remove products in his cart

### PR DESCRIPTION
**Pull request by huyhoangdayhoc**
Thực hiện Issues27463 : 

See Cart, Update Quantity, Remove products in cart
Screenshot from 2020-08-27 11-13-05

Select quantity và Add to Cart
Screenshot from 2020-08-27 11-14-35

### CODE REVIEW

**Comment by huyhoangdayhoc** [comment]
ready

File: resources/views/pages/components/cart.blade.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                    <h3 class="title">Order</h3>
+                </div>
+                    <table class="shopping-cart-table table">
+                        <thead>
+                            <tr>
+                                <th class="text-center">Image</th>
+                                <th class="text-center">Name</th>
+                                <th class="text-center">Price</th>
+                                <th class="text-center">Quantity</th>
+                                <th class="text-center">Total</th>
```
Comment (quoctd-0607) [inline resources/views/pages/components/cart.blade.php]: i18n

File: resources/views/pages/components/navigation.blade.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                        @endif
+                    </ul>
+                </div>
+            </div>
+            <div class="col-lg-9">
+                <div class="hero__search">
+                    <div class="hero__search__form">
+                        <form action="#">
+                            <div class="hero__search__categories">
+                                All Categories
```
Comment (quoctd-0607) [inline resources/views/pages/components/navigation.blade.php]: i18n. fix hết nhé e

File: resources/views/pages/components/cart.blade.php (hunk 1 above)
Comment (huyhoangdayhoc) [reply resources/views/pages/components/cart.blade.php]: dạ, em check mà quên mất

File: resources/views/pages/components/navigation.blade.php (hunk 2 above)
Comment (huyhoangdayhoc) [reply resources/views/pages/components/navigation.blade.php]: dạ

**Comment by huyhoangdayhoc** [comment]
ready

**Comment by huyhoangdayhoc** [comment]
ready 

fix conflict

**Pull request merged**